Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -28,6 +28,7 @@
 #include "APIObject.h"
 #include "MessageReceiver.h"
 #include "MessageSender.h"
+#include "ProcessThrottler.h"
```

Comment:
Well, that is kind of defeating the purpose of extracting TimedActivity out of ProcessThrottler just so that you can forward declare it below.

Reply:
Is there a way to declare a `TimedActivity` property without the full definition from the header? I was using a `UniqueRef` to avoid this but I agree that a heap allocation is not optimal. If this is the only way I'll remove the forward declarations revert extracting `TimedActivity`